scripts/requirements.txt

```diff
@@ -1,3 +1,28 @@
-SPARQLWrapper
-rdflib
-requests
\ No newline at end of file
+antlr4-python3-runtime==4.9.1
+brotlipy==0.7.0
+certifi==2020.12.5
+cffi @ file:///home/conda/feedstock_root/build_artifacts/cffi_1606601120025/work
```

Comment:
@NuriaQueralt I get errors on installing these packages due to local path. Is it possible to remove unused packages?

Reply:
Hi @rajaram5 requested changes on requirements.txt done. I have also renamed validation scripts from *Syntaxic* to *Syntax*.